gizmos.py

```diff
@@ -225,6 +230,8 @@ def draw(self, context):
 
     def draw_select(self, context, select_id):
```

Comment:
I assume the gizmos which define the test_select function should also check the visibility first.

Reply:
Yeah i wasn't able to actually select it when it's not drawn, strange...